src/refiners/foundationals/clip/image_encoder.py

```diff
@@ -126,6 +128,7 @@ def __init__(
         self.num_layers = num_layers
         self.num_attention_heads = num_attention_heads
         self.feedforward_dim = feedforward_dim
+        select_first_embedding: Callable[[Tensor], Tensor] = lambda x: x[:, 0, :]
```

Comment:
nit: I feel like `cls_pooling` (or `cls_token_pooling`) would be even more expressive here